builtin/logical/transit/path_sign_verify.go

```diff
@@ -43,6 +43,21 @@ derivation is enabled; currently only available with ed25519 keys.`,
 * sha2-384
 * sha2-512
 
+Defaults to "sha2-256". Not valid for all key types,
+including ed25519.`,
+			},
+
+			"algorithm": &framework.FieldSchema{
+				Type:    framework.TypeString,
+				Default: "sha2-256",
+				Description: `Deprecated: use "hash_algorithm" instead. Hash algorithm to use (POST body parameter). Valid values are:
```

Comment:
I don't think you need to duplicate the help text here; should be sufficient to just mark it deprecated and to use hash_algorithm instead, since they can see the help text there, and the behavior for both parameters is exactly the same.